CHANGELOG.md

```diff
@@ -1,152 +1,84 @@
-- Fixed an issue where the Kui nameplates customization wouldn't properly hide friendly nameplates according to the settings under specific circumstances.
-- Fixed mount and character tooltips overlapping in some cases when the tooltip is anchored on the cursor.
-
-# Changelog version 2.8.2
-
-*The rate limit changes mentioned last version no longer apply in the outdoor world or garrisons. Instances are still heavily limited.*
-
-## Added
-
-- Added "Open Profile" right-click menu option for pets and battle pets.
-- Added 94 icons and 47 images from patch 10.2.7 (and earlier).
+This version adds support for patch 11.0.
 
-## Fixed
-
-- Fixed battle pets not being able to get a profile assigned on Cataclysm Classic.
-  - As part of this fix, it is now possible to attach a profile to a non-renamed battle pet on Retail. It is still strongly advised to rename battle pets before assigning them a profile.
-- Fixed map scan not working if the player was in Telogrus Rift.
-- Fixed loading icon sometimes showing on companion tooltips.
-
-# Changelog version 2.8.1
-
-*Profile transfers will currently be much slower on Cataclysm Classic due to rate limit changes. We are looking into changes to alleviate this, but they will take time, please bear with us.*
-
-## Fixed
-
-- Fixed icons not showing in the tooltips on Classic.
-- Fixed the "receiving profile" animated icon appearing as a green square on Classic.
-- Fixed an issue when editing profiles with an invalid icon (for example, a Retail icon on Classic).
-- Pasting a color code into the color picker field will now trim spaces that might have been copied with it to prevent issues.
-
-# Changelog version 2.8.0
+Thanks to Raenore for the help on some of these features, and keyboardturner for the new silhouette logo.
 
 ## Added
-
-- Added a small indicator at the bottom of the tooltip while receiving updates to a profile.
+- Added a new Relations menu located in the Customization section, alongside the moved Automation settings.
+  - This menu lets you edit the preset relations you can have with other players, as well as create new ones.
+  - Those relations are still only personal, and not shared with other players.
+  - The relation will now be shown as a line on the tooltip in addition to the border color. This can be changed to show the relation description, or disabled in Tooltip settings.
+- Added a "Currently" toolbar button letting you open a frame to quickly modify your Currently info.
+- Added a toolbar button to open/close the main Total RP 3 window.
+- Added a target frame button for your own companion to auto-fill the speech window with its name in brackets.
+  - This will apply RP name customization to its speech lines.
+- Added a dropdown to set whether you are open to other players walking up to your character.
+  - This will show as an icon on your tooltip.
+  - Existing tooltip bottom icons have also been changed for clarity.
+    - **TODO: INSERT IMAGE**
```

Comment:
```suggestion
  - This will show as an icon on your tooltip and in your profile characteristics.
  - Existing tooltip bottom icons have also been changed for clarity.
    ![image](https://github.com/user-attachments/assets/2ed353da-588a-45c4-b7af-fc8dfa7b9c74)
    - Book: Profile has an unread description.
    - Note: You have notes on this profile.
    - Three people: Walkup friendly.
```

This hopefully takes care of that, img below too:
![image](https://github.com/user-attachments/assets/2ed353da-588a-45c4-b7af-fc8dfa7b9c74)